README.md

```diff
@@ -1,2 +1,5 @@
-# technical-documentation
-A technical documentation template web app...
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Kindly remove unnecessary details in your README.md and make it looks more professional. Check out this [repo](https://github.com/Ndaboom/portofolio) for an overview of what it's supposed to look like😋